common/lib/xmodule/xmodule/js/src/video/035_video_accessible_menu.js

```diff
@@ -218,14 +230,24 @@ function () {
                     _closeMenu(this);
                     break;
                 // Close menu and give focus to speed control.
+
                 case KEY.ESCAPE:
                     _closeMenu(this);
                     this.button.focus();
                     break;
             }
             return false;
-        }
-        else {
+
+        } else if (target.is('.has-dropdown')) {
+
+            switch(keyCode) {
```

Comment:
I didn't write this script, I just updated it. Want me to go through and make these adjustments?